source/includes/kubernetes/_k8_secrets.md

```diff
@@ -61,33 +61,112 @@ curl -X GET \
+      "namespace": "a3ViZS1zeXN0ZW0=",
+      "token": "ZXlKaGJHY2lPaUpTVXpJMU5pSXNJbXRwWkNJNklpSjkuZXlKcGMzTWlPaUpyZFdKbGNtNWxkR1Z6TDNObGNuWnBZMlZoWTJOdmRXNTBJaXdpYTNWaVpYSnVaWFJsY3k1cGJ5OXpaWEoyYVdObFlXTmpiM1Z1ZEM5dVlXMWxjM0JoWTJVaU9pSnJkV0psTFhONWMzUmxiU0lzSW10MVltVnlibVYwWlhNdWFXOHZjMlZ5ZG1salpXRmpZMjkxYm5RdmMyVmpjbVYlsdkjfkdfwTG01aGJXVWlPaUpqWVc1aGJDMTBiMnRsYmkwMk56bG9OeUlzSW10MVltVnlibVYwWlhNdWFXOHZjMlZ5ZG1salpXRmpZMjkxYm5RdmMyVnlkbWxqWlMxaFkyTnZkVzUwTG01aGJXVWlPaUpqWVc1aGJDSXNJbXQxWW1WeWJtVjBaWE11YVc4dmMyVnlkbWxqWldGalkyOTFiblF2YzJWeWRtbGpaUzFoWTJOdmRXNTBMblZwWkNJNkltUTVOamcyWVRWbExUUTBaak10TVRGbE9TMDVOakJrTFRBeU1EQTNZVGxoTURBeFppSXNJbk4xWWlJNkluTjVjM1JsYlRwelpYSjJhV05sWVdOamIzVnVkRHByZFdKbExYTjVjM1JsYlRwallXNWhiQ0o5LkN2LTZiWFdjQlU1VnZCZjJKSUVpc1BIOHZKYTllY01ra0JNbTg0dE9Qd3N1MVNnWUp3V2cwVlBFT2hnTzltdUNoMzFMTzRMOHpabklaeU0yUUNoRmo4TEtKcDJJU2FXa05Nb0c4RmJUcEtCWGNKd2FOZVNCMlo2M01LRnNyZlhQMnpkSmFjSGVuUi1KaTJsdW5rMmFRYnE3eFVreGxqdkc5SVVEY05aZmlrUllhbUVhd1hFeG1RYVhkalpGRWgwT2J2VDVsdGVVY2xYcXhIMFpzWktUNWRfYWttRmRZT3B3UVZOSUlDS0JxZmFZWjBULXliMmxmbDREYTFNUndNLV93TzVXNUdtbVo1UGdubGQxTjZqaGxEekladWp6cVFQSlh3MGU4V3VXVVlPX09YNDJ5YVVoM3pUS3F1ejhVTjVkS0pDcEF4WFd4RWFsZWV0MThMUFUwUQ=="
+    },
+    "kind": "Secret",
+    "metadata": {},
+    "type": "kubernetes.io/service-account-token"
+  }
 }
 ```
 
 <code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/secrets/:id</code>
 
 Retrieve a secret and all its info in a given [environment](#administration-environments).
 
-| Attributes                 | &nbsp;                                                                     |
-| -------------------------- | ---------------------------------------------------------------------------|
-| `id` <br/>_string_         | The id of the secret.                                                      |
-| `apiVersion` <br/>_string_ | The API version used to retrieve the secret.                               |
-| `data` <br/>_string_       | The data that the secret contains.                                         |
-| `metadata` <br/>_object_   | The metadata of the secret.                                                |
-| `kind` <br/>_string_       | A string value representing the REST resource that this object represents. |
-| `type` <br/>_string_       | A string used to facilitate programmatic handling of a secret's data.      |
+| Attributes                  | &nbsp;                                                                     |
+| --------------------------- | -------------------------------------------------------------------------- |
+| `id` <br/>_string_          | The id of the secret.                                                      |
+| `apiVersion` <br/>_string_  | The API version used to retrieve the secret.                               |
+| `encodedData` <br/>_string_ | The data that the secret contains.                                         |
```

Comment:
Should we also mention which encoding is used for `encodedData` as done in k8s docs?

Reply:
a855197d879b83a378f7bb703864deb33ea8ed53